src/app/tab/covid/covid.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="covidContainer" [style.padding]="getWidth(settings.config.covidData.countries[index].padding)">
```

Comment:
The padding option should adjust the "gap" between flex items, in this case, the space between the flag and the text. You can use the CSS property "gap" here. A margin will be used instead to space between other elements, I mention this in another comment.